### PR TITLE
feat: APPS-1970 Create listing page for Endowments

### PR DESCRIPTION
**Pull request by dianneweinthal**
Connected to [APPS-1970](https://jira.library.ucla.edu/browse/APPS-1970)

Endowment listing

-   [x] I added github label for semantic versioning
-   [x] I double checked it looks like the designs
-   [ ] I completed any required mobile breakpoint styling
-   [ ] I completed any required hover state styling
-   [ ] I included a working spec file
-   [ ] I added notes above about how long it took to build this component
-   [ ] UX has reviewed this PR
-   [x] I assigned this PR to someone to review

### CODE REVIEW

**Review by jendiamond — APPROVED** [review]

File: pages/support/endowments/index.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            @search-ready="getSearchData"
+        /> -->
+
+        <section-wrapper theme="divider">
+            <divider-way-finder
+                color="about"
+            />
+        </section-wrapper>
+
+        <section-wrapper>
```
Comment (jendiamond) [inline pages/support/endowments/index.vue:29]: The links on the titles are going to `/undefined`

File: pages/support/endowments/index.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<template lang="html">
+    <main
+        class="page page-endowment-listing"
+    >
+        <nav-breadcrumb
```
Comment (jendiamond) [inline pages/support/endowments/index.vue:5]: Is the Support Us bread crumb is going to the correct place? `/support`

**Review by jendiamond — CHANGES_REQUESTED** [review]

File: pages/support/endowments/index.vue (hunk 2 above)
Comment (jendiamond) [reply pages/support/endowments/index.vue:5]: yes, this is correct

File: pages/support/endowments/index.vue (hunk 1 above)
Comment (dianneweinthal) [reply pages/support/endowments/index.vue:29]: updated query

**Review by jendiamond — APPROVED** [review]
🍰

**Pull request merged**